### PR TITLE
[BUGFIX] No longer refer to removed hooks for preview

### PR DESCRIPTION
**Pull request by sypets**
Resolves: #538

### CODE REVIEW

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 1]
```diff
@@ -298,14 +298,18 @@ tt_content.preview
...
 
-        ..  code-block:: php
-            :caption: Search for registrations of this hook
+        Have a look at :php:`\TYPO3\CMS\Backend\Preview\StandardContentPreviewRenderer`
+        and the various methods for customizing the preview rendering:
 
-            $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['cms/layout/class.tx_cms_layout.php']
-                ['tt_content_drawItem']['content_element_xy'];
+        #. :ref:`Register a preview renderer <t3coreapi:ConfigureCE-Preview>`
+        #. Register an event listener for :ref:`PageContentPreviewRenderingEvent <t3coreapi:PageContentPreviewRenderingEvent>`
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:309]:
```suggestion
        #.  :ref:`Register a preview renderer <t3coreapi:ConfigureCE-Preview>`
        #.  Register an event listener for :ref:`PageContentPreviewRenderingEvent <t3coreapi:PageContentPreviewRenderingEvent>`
```

**Review by froemken — APPROVED** [review]
Looks good to me

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 2]
```diff
@@ -298,14 +298,18 @@ tt_content.preview
...
+        Have a look at :php:`\TYPO3\CMS\Backend\Preview\StandardContentPreviewRenderer`
+        and the various methods for customizing the preview rendering:
 
-            $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['cms/layout/class.tx_cms_layout.php']
-                ['tt_content_drawItem']['content_element_xy'];
+        #.  :ref:`Register a preview renderer <t3coreapi:ConfigureCE-Preview>`
+        #.  Register an event listener for :ref:`PageContentPreviewRenderingEvent <t3coreapi:PageContentPreviewRenderingEvent>`
+
+        For checking for event listeners, see the list
+        :guilabel:`System > Configuration > Event Listeners (PSR-14)`.
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:312]:
```suggestion
        :guilabel:`System > Configuration > Event Listeners (PSR-14)` (available with installed EXT:lowlevel).
```

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@sypets Thanks for the improvement :+1:

**Pull request merged**